xdsl/interpreters/eqsat_pdl_interp.py

```diff
@@ -26,6 +33,16 @@
 from xdsl.utils.scoped_dict import ScopedDict
 
 
+@dataclass
+class NonPropagatingDataFlowSolver(DataFlowSolver):
+    propagate: bool = True
```

Comment:
```suggestion
    propagate: bool = field(default=True)
```
Is it worth being consistent on these? (Genuine question, I am not sure)